lib/vdc/BOVCollection.cpp

```diff
@@ -0,0 +1,369 @@
+#include <vector>
+#include <algorithm>
+#include <map>
+#include <sstream>
+#include <iomanip>
+#include <iostream>
+#include <fstream>
+#include <type_traits>
+#include "vapor/VAssert.h"
+#include <stdio.h>
+
+#ifdef _WINDOWS
+    #define _USE_MATH_DEFINES
+    #pragma warning(disable : 4251 4100)
+#endif
+#include <cmath>
+
+#include <vapor/BOVCollection.h>
+
+using namespace VAPoR;
+
+const std::string BOVCollection::_timeToken = "TIME";
+const std::string BOVCollection::_dataFileToken = "DATA_FILE";
+const std::string BOVCollection::_dataSizeToken = "DATA_SIZE";
+const std::string BOVCollection::_formatToken = "DATA_FORMAT";
+const std::string BOVCollection::_variableToken = "VARIABLE";
+const std::string BOVCollection::_endianToken = "DATA_ENDIAN";
+const std::string BOVCollection::_centeringToken = "CENTERING";
+const std::string BOVCollection::_originToken = "BRICK_ORIGIN";
+const std::string BOVCollection::_brickSizeToken = "BRICK_SIZE";
+const std::string BOVCollection::_offsetToken = "BYTE_OFFSET";
+const std::string BOVCollection::_divideBrickToken = "DIVIDE_BRICK";
+const std::string BOVCollection::_dataBrickletsToken = "DATA_BRICKLETS";
+const std::string BOVCollection::_dataComponentsToken = "DATA_COMPONENTS";
+
+const std::string BOVCollection::_xDim = "x";
+const std::string BOVCollection::_yDim = "y";
+const std::string BOVCollection::_zDim = "z";
+const std::string BOVCollection::_timeDim = "t";
+
+const std::string BOVCollection::_byteFormatString = "BYTE";
+const std::string BOVCollection::_shortFormatString = "SHORT";
```

Comment:
These 2 variables are declared, but not used anywhere else. Would it make sense to delete them?

Reply:
For completeness, I'd like to keep all data type identifiers defined in the BOV specification.  This will save me from having to look up the question "what data types are supported in BOV, and how are they defined?" in the future.